cmd/gmailctl/cmd/init_cmd.go

```diff
@@ -112,3 +122,104 @@ func createDefault(path, contents string) (err error) {
 	}
 	return nil
 }
+
+func resetLib() (err error) {
+	libFile := path.Join(cfgDir, "gmailctl.libsonnet")
+	libFileOld := path.Join(cfgDir, "gmailctl.libsonnet.backup")
+
+	// check for existence of current libFile
+	if _, err := os.Stat(libFile); err == nil {
+		// rsection for moving/renaming old file as backup
+		err := os.Rename(libFile, libFileOld)
+		if err != nil {
+			// probably a permissions error?
+			return err
+		}
+	} else if !os.IsNotExist(err) {
+		// something weird
+		return err
+	} else {
+		// it just simply doesn't exist, so we can proceed
+	}
+
+	err = createDefault(libFile, data.GmailctlLib())
+
+	if err != nil {
+		// probably a permissions error?
+		return err
+	}
+
+	//compare the old and new files to avoid making an identical backup
+	// use MD5 to be fast; security not a concern
```

Comment:
We don't need this, because we're just going to override the file without backups.